domain/status/state/state_test.go

```diff
@@ -50,8 +51,8 @@ func (s *stateSuite) SetUpTest(c *gc.C) {
 	modelUUID := uuid.MustNewUUID()
 	err := s.TxnRunner().StdTxn(context.Background(), func(ctx context.Context, tx *sql.Tx) error {
 		_, err := tx.ExecContext(ctx, `
-			INSERT INTO model (uuid, controller_uuid, name, type, cloud, cloud_type)
-			VALUES (?, ?, "test", "iaas", "test-model", "ec2")
+			INSERT INTO model (uuid, controller_uuid, name, type, cloud, cloud_type, credential_owner)
+			VALUES (?, ?, "test", "iaas", "test-model", "ec2", "owner")
 		`, modelUUID.String(), coretesting.ControllerTag.Id())
```

Comment:
We don't want to use core testing here. I would suggest wither just generating a new uuid for the controller uuid or hardcoding one that you generate in the sql here.

Reply:
Sure